NickvisionMoney.GNOME/Views/MainWindow.cs

```diff
@@ -490,6 +495,16 @@ private void KeyboardShortcuts(Gio.SimpleAction sender, EventArgs e)
         shortcutsDialog.Show();
     }
 
+    /// <summary>
+    /// Occurs when quit action is triggered
+    /// </summary>
+    /// <param name="sender">Gio.SimpleAction</param>
+    /// <param name="e">EventArgs</param>
+    private void Quit(Gio.SimpleAction sender, EventArgs e)
```

Comment:
Single-lined methods in C# are usually condensed into one line. Could you make this:
```csharp
private void Quit(Gio.SimpleAction sender, EventArgs e) => _application.Quit();
```

Reply:
Not all that familiar with C#, TIL :slightly_smiling_face: